README.md

```diff
@@ -15,6 +15,7 @@ Bottlerocket-specific additions focus on reliable updates and on the API.
 Instead of making configuration changes manually, you can change settings with an API call, and these changes are automatically migrated through updates.
 
 Some notable features include:
+
```

Comment:
the linter in my editor (vscode) added all these newlines any time I saved the file -- if this is benign, I'll leave them in.

Reply:
Please leave them out of this PR and make a separate commit and PR it separately.